doc/source/how-to-run-flower-using-docker.rst

```diff
@@ -245,6 +264,142 @@ certificate within the container. Use the ``--certificates`` flag when starting
     --server 192.168.1.100:9092 \
     --certificates ca.crt
 
+Flower ServerApp
+----------------
+
+The procedure for building and running a ServerApp image is almost identical to the SuperNode image.
+A key difference is the additional argument in the ``ENTRYPOINT`` command of the ServerApp
+Dockerfile.
+
+Similar to the SuperNode image, the ServerApp Docker image comes with a pre-installed version of
+Flower and serves as a base for building your own ServerApp image.
+
+We will use the same ``app-pytorch`` example as we do in the Flower SuperNode section.
+If you have not already done so, please follow the `SuperNode Prerequisites`_ before proceeding.
+
+
+Creating a ServerApp Dockerfile
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Let's assume the following project layout:
+
+.. code-block:: bash
+
+  $ tree .
+  .
+  ├── server.py        # ServerApp code
+  ├── task.py          # ServerApp code
+  ├── requirements.txt # ServerApp dependencies
```

Comment:
In the `examples/app-pytorch` I need the`requirements.txt` because the ServerApp code uses the pytorch package but we can create our own example. wdyt?

Reply:
Yes, let's adapt the examples. In fact, the `quickstart-pytorch` example is a better example to base this "How-to" on. For context, the `app-pytorch` uses a server-side model parameter intialization - that's why PyTorch is a dependency for running the `ServerApp`. For the `quickstart-pytorch`, it is a client-side parameter initialization - so all ML-related computation is done in the `ClientApp`. Can you please update the references to `quickstart-pytorch` instead?

Reply:
Oh I didn't know that. That makes sense! I will update it to use the quickstart-pytorch example instead.